doc/encoding.rst

```diff
@@ -50,6 +50,32 @@ categories, eg to replace :class:`~sklearn.preprocessing.OneHotEncoder`:
        Similarity encoding for learning with dirty categorical variables. 2018.
        Machine Learning journal, Springer.
 
+
+Encoding text with diverse entries
+----------------------------------
+
+When it comes to encoding text with diverse entries, large language models
```

Comment:
"large language model" might have users thinking "i can't run this on my machine", maybe insist on the fact that there are language models of all shapes and sizes on the hugging face hub and that now thanks to skrub you can plug any of them in a scikit-learn pipeline for tabular data